### PR TITLE
replace var with let/const where appropriate.

### PR DESCRIPTION
**Pull request by dayopius**
Fixes #1086 (<=== Add issue number here)

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [x] PR is descriptively titled 📑 and links the original issue above 🔗
* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `grunt test`
* [x] code is in uniquely-named feature branch and has no merge conflicts 📁
* [x] screenshots/GIFs are attached 📎 in case of UI updates
* [x] @mention the original creator of the issue in a comment below for help or for a review

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software 

Thanks!

### CODE REVIEW

**Comment by dayopius** [comment]
@TildaDares

**Review by TildaDares — APPROVED** [review]
Great work @dayopius. Thank you!!

File: src/edit/actions/DistortAction.js [hunk 1]
```diff
@@ -3,17 +3,17 @@ L.DistortAction = L.EditAction.extend({
...
       svg: true,
-      html: 'distort',
+      html: "distort",
       tooltip: overlay.options.translation.distortImage,
-      className: 'distort',
+      className: "distort",
     };
 
-    L.DistortableImage.action_map.d = '_distortMode';
+    L.DistortableImage.action_map.d = "_distortMode";
```
Comment (TildaDares) [inline src/edit/actions/DistortAction.js:11]:
```suggestion
    L.DistortableImage.action_map.d = '_distortMode';
```

File: src/edit/actions/DistortAction.js [hunk 2]
```diff
@@ -3,17 +3,17 @@ L.DistortAction = L.EditAction.extend({
     options = options || {};
     options.toolbarIcon = {
       svg: true,
-      html: 'distort',
+      html: "distort",
       tooltip: overlay.options.translation.distortImage,
-      className: 'distort',
+      className: "distort",
```
Comment (TildaDares) [inline src/edit/actions/DistortAction.js:8]:
```suggestion
      className: 'distort',
```

File: src/edit/actions/DistortAction.js [hunk 3]
```diff
@@ -3,17 +3,17 @@ L.DistortAction = L.EditAction.extend({
     options = options || {};
     options.toolbarIcon = {
       svg: true,
-      html: 'distort',
+      html: "distort",
```
Comment (TildaDares) [inline src/edit/actions/DistortAction.js:6]:
```suggestion
      html: 'distort',
```

**Review by TildaDares — CHANGES_REQUESTED** [review]
Feel free to just click the `Commit Suggestion` buttons @dayopius

**Review by TildaDares — APPROVED** [review]
Amazing work @dayopius. Thank you!

**Pull request merged**